app/errors.js

```diff
@@ -5,3 +5,12 @@ const internalError = (message, internalCode) => ({
 
 exports.DEFAULT_ERROR = 'default_error';
 exports.defaultError = message => internalError(message, exports.DEFAULT_ERROR);
+
+exports.EMAIL_DUPLICATED = 'email_duplicated';
+exports.emailDuplicated = email => internalError(`email duplicated: ${email}`, exports.EMAIL_DUPLICATED);
+
+exports.EMAIL_NOT_VALID = 'email_not_valid';
+exports.emailNotValid = email => internalError(`email not valid: ${email}`, exports.EMAIL_NOT_VALID);
+
+exports.PASSWORD_NOT_VALID = 'password_invalid';
+exports.passwordInvalid = () => internalError('Password invalid', exports.PASSWORD_NOT_VALID);
```

Comment:
You could also define this like a value instead of a function with:
`exports.passwordInvalid = internalError('Password invalid', exports.PASSWORD_NOT_VALID);`
and after that in the controller you can use it with
`errors.passwordInvalid` and it would work fine.